src/components/CanvasWrapper.tsx

```diff
@@ -616,12 +650,32 @@ export default function CanvasWrapper(inputProps: CanvasWrapperProps): ReactElem
       </span>
     )
   );
+  const labelData: LabelData | undefined = labels[props.annotationState.currentLabelIdx ?? 0];
+  const shouldShowReuseValueHotkey = labelData?.options.type === LabelType.INTEGER && labelData?.options.autoIncrement;
```

Comment:
I changed the popup in the corner to include shortcut keys. The visible keys are context-dependent, so the `Ctrl` hotkey is only shown if the current label automatically changes values (e.g. auto-incrementing integer labels)

![image](https://github.com/user-attachments/assets/2ccfb22b-54fc-4910-aae3-c5f1e852f40e)